sonar-project.properties

```diff
@@ -0,0 +1,26 @@
+# WARNING - THIS FILE WAS GENERATED BY THE dps-gradle-spring-boot GRADLE PLUGIN
```

Comment:
auto generated from spring boot plugin.  Could have added to `.gitignore` since not scanned by cloud platform :shrug: